easybuild/tools/toolchain/toolchain.py

```diff
@@ -333,10 +333,13 @@ def prepare(self, onlymod=None):
         if not self._toolchain_exists():
             self.log.error("No module found for toolchain: %s" % self.mod_short_name)
 
-        if self.name == DUMMY_TOOLCHAIN_NAME:
+        
+	if self.name == DUMMY_TOOLCHAIN_NAME:
             if self.version == DUMMY_TOOLCHAIN_VERSION:
                 self.log.info('prepare: toolchain dummy mode, dummy version; not loading dependencies')
-            else:
+		#@todo I keep this is as, but want dummy to do the same as a regular toolchain. Need to think this through.
```

Comment:
at the very least, also cleanup the log message.
wrt the todo, any progress? i think this requires an extra option to have this behaviour. (as it's very backwards incompatible).